sim/deathknight/ghoul_pet_abilities.go

```diff
@@ -61,8 +62,9 @@ func (ghoulPet *GhoulPet) newClaw() PetAbility {
 
 			Cast: core.CastConfig{
 				DefaultCast: core.Cast{
-					GCD: core.GCDDefault,
+					GCD: time.Second,
```

Comment:
isn't it better to use `core.GCDMin` which is 1 second

Reply:
I've gone with the rogue house-style, that's simply using `time.Second`. `core.GCDMin` is essentially a "casts affected by cast speed limit", and doesn't quite apply here.